index.html

```diff
@@ -5,6 +5,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <h1>Students list</h1>
+    <li><a href="/Full stack/projects/netcraft-11-2022/DavidLevy/index.html">Lecture - David Levy</a></li>
```

Comment:
`./DavidLevy/index.html`
i think this is the right path

Reply:
thx.